src/main/jbake/content/hawkular-services/docs/installation-guide/secure-comm.adoc

```diff
@@ -72,6 +72,13 @@ First, you must add a security-realm in standalone/configuration/standalone.xml:
         </ssl>
       </server-identities>
     </security-realm>
+    <security-realm name="HawkularAgentRealm">
+      <server-identities>
+        <ssl>
+          <truststore path="hawkular.keystore" relative-to="jboss.server.config.dir" keystore-password="hawkular" />
+        </ssl>
+      </server-identities>
+    </security-realm>
 ----
```

Comment:
Actually i don't think is similar, Truststore must be placed in authentication node as far i remeber.